client/state/media/reducer.js

```diff
@@ -454,6 +455,13 @@ export const googlePhotosPicker = ( state = {}, action ) => {
 				session: action.session,
 			};
 		}
+
+		case MEDIA_PHOTOS_PICKER_FEATURE_FLAG_SET: {
```

Comment:
Should we specify this as something like `MEDIA_GOOGLE_PHOTOS_PICKER_FEATURE_FLAG_SET` to be more specific here? If we want to keep this general, maybe `featureEnabled` can be more specific, what do you think?

Reply:
It makes sense, but I followed the other action name to be consistent.
- `MEDIA_PHOTOS_PICKER_SESSION_SET` and
- `MEDIA_PHOTOS_PICKER_FEATURE_FLAG_SET`